[2024-06-24T09:48:02.538+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T09:48:02.561+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-24T09:47:00+00:00 [queued]>
[2024-06-24T09:48:02.572+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-24T09:47:00+00:00 [queued]>
[2024-06-24T09:48:02.573+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-24T09:48:02.591+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-24 09:47:00+00:00
[2024-06-24T09:48:02.596+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2208) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T09:48:02.598+0000] {standard_task_runner.py:63} INFO - Started process 2210 to run task
[2024-06-24T09:48:02.598+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-24T09:47:00+00:00', '--job-id', '3557', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmpzar7jp49']
[2024-06-24T09:48:02.599+0000] {standard_task_runner.py:91} INFO - Job 3557: Subtask check_previous_run_status
[2024-06-24T09:48:02.647+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-24T09:47:00+00:00 [running]> on host bf895605831d
[2024-06-24T09:48:02.748+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T09:47:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T09:47:00+00:00'
[2024-06-24T09:48:02.748+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T09:48:02.761+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-24T09:48:02.761+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-24T09:48:02.761+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-24T09:48:02.761+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-24T09:48:02.776+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-24T09:48:02.796+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T09:48:02.815+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-24T09:47:00+00:00, execution_date=20240624T094700, start_date=20240624T094802, end_date=20240624T094802
[2024-06-24T09:48:02.851+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T09:48:02.891+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T09:48:02.893+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
